packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -2316,83 +3389,122 @@ test.serial('writeGranulesFromMessage() honors granule.createdAt time if provide
     testOverrides: { stepFunctionUtils },
   });
 
-  const pgGranule = await t.context.granulePgModel.get(
-    knex,
-    { granule_id: granuleId, collection_cumulus_id: collectionCumulusId }
+  t.true(
+    await granulePgModel.exists(knex, {
+      granule_id: completeGranule.granuleId,
+      collection_cumulus_id: collectionCumulusId,
+    })
   );
-  t.is(pgGranule.created_at.getTime(), expectedCreatedAt);
-});
-
-test.serial('writeGranulesFromMessage() throws if workflow_start_time is not provided on the message', async (t) => {
-  const {
-    cumulusMessage,
-    knex,
-    executionCumulusId,
-    providerCumulusId,
-    stepFunctionUtils,
-  } = t.context;
-
-  delete cumulusMessage.cumulus_meta.workflow_start_time;
+  t.true(await esGranulesClient.exists(completeGranule.granuleId));
 
-  await t.throwsAsync(writeGranulesFromMessage({
-    cumulusMessage,
-    executionCumulusId,
-    providerCumulusId,
-    knex,
-    testOverrides: { stepFunctionUtils },
-  }), { instanceOf: CumulusMessageError });
-});
+  const initialPostgresRecord = await granulePgModel.get(knex, {
+    granule_id: granuleId,
+    collection_cumulus_id: collectionCumulusId,
+  });
+  const apiFormattedInitialPostgresGranule =
+    await translatePostgresGranuleToApiGranule({
+      granulePgRecord: initialPostgresRecord,
+      knexOrTransaction: knex,
+    });
 
-test.serial('writeGranulesFromMessage() falls back to workflow_start_time if granule.createdAt is not provided in cumulus_message for a granule', async (t) => {
```

Comment:
Oof, this diff is rough - confirmed this is just a down-file move.

Reply:
yeah